src/sharing/participant.rs

```diff
@@ -184,6 +184,34 @@ impl<G: Group> ActiveParticipant<G> {
         );
         (VerifiableDecryption::from_element(dh_element), proof)
     }
+
+    /// Takes decryption shares and combines them together with a ciphertext to obtain
+    /// the encrypted element. Returns None, if there is not enough shares.
+    pub fn finish_decryption(&self, shares: Vec<(usize, VerifiableDecryption<G>)>, ciphertext: Ciphertext<G>) -> Option<G::Element> {
```

Comment:
Could you please explain a use case where this method will be used? It seems to be a redundant copy of [`Params::combine_shares()`](https://slowli.github.io/elastic-elgamal/elastic_elgamal/sharing/struct.Params.html#method.combine_shares), given that this method only uses `Params` from `self`, and `Params` are publicly accessible via `self.key_set().params()`.

Reply:
You are right; it indeed is redundant. I somehow managed to miss the 'combine_shares()' method.